src/trafficcontroller/dopplerproxy/doppler_proxy.go

```diff
@@ -137,13 +137,24 @@ func (proxy *Proxy) serveAppLogs(paths []string, writer http.ResponseWriter, req
 		return
 	}
 
-	authorized, err := proxy.logAuthorize(authToken, appId, proxy.logger)
+	status, _ := proxy.logAuthorize(authToken, appId, proxy.logger)
```

Comment:
Most of the errors are logged within the logAuthorized except one path. Passing the error back and logging it again is just duplicate noise.